src/index.js

```diff
@@ -168,18 +174,33 @@ const setLang = (lng) => {
       if (ddb.isLoaded()) {
         drawKpis(ddb.totals, ddb.totalsDiff, LANG);
         drawPageTitleCount(ddb.totals.confirmed, LANG);
-        trendChart = drawTrendChart(
-          ddb.trend,
-          trendChart,
-          LANG,
-          CHART_TIME_PERIOD
-        );
-        dailyIncreaseChart = drawDailyIncreaseChart(
-          ddb.trend,
-          dailyIncreaseChart,
-          LANG,
-          CHART_TIME_PERIOD
-        );
+
+        // TODO: This doesn't seem to be needed and can be removed.
```

Comment:
Maybe it's safe to remove this too

Reply:
I'm thinking of removing this in a separate change, I'm trying to figure out if there's a corner case we used to have which we kept this around for.